client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -13,22 +13,128 @@ function isNumberField(field) {
 
 const COLUMN_WIDTHS = {
```

Comment:
maybe we should do width types, like XS, S, M, L, and assign width via CSS classes